lib/nimble_template/addons/variants/phoenix/api/empty_body_plug.ex

```diff
@@ -0,0 +1,54 @@
+defmodule NimbleTemplate.Addons.Phoenix.Api.EmptyBodyPlug do
```

Comment:
Can we rename it the same as the file name? `EmptyBodyParamsPlug`

Reply:
good catch, updated in https://github.com/nimblehq/elixir-templates/pull/164/commits/43577587fd1e055db9901b8bd2ec2d1f119c2fb3